src/Hazelcast.Net/Core/TaskCoreExtensions.WithCancellation.cs

```diff
@@ -23,8 +23,8 @@ namespace Hazelcast.Core
 {
     internal static partial class TaskCoreExtensions
     {
-        public static ConfiguredCancelableAsyncEnumerable<T> WithCancellation<T>(this IAsyncEnumerable<T> enumerable, CancellationToken cancellationToken, bool throwOnCancel)
-            => new ConfiguredCancelableAsyncEnumerable<T>(enumerable, continueOnCapturedContext: true, cancellationToken, throwOnCancel);
+        public static ConfiguredCancelableAsyncEnumerable<T> WithCancellation<T>(this IAsyncEnumerable<T> enumerable, bool throwOnCancel, CancellationToken cancellationToken)
```

Comment:
I am not really familiar with C#, it says internal at the top but I saw that we use `WithCancellation` in tests like `src/Hazelcast.Net.Tests/Sql/SqlQueryResultTests.cs`. So, if it is just an internal thing, please ignore this comment.

But if this API is public, can the change on the parameter order be a problem?

Reply:
It could. But this is a fix to code that we haven't released yet. And, yes, the class is internal so users won't see it.